janusgraph-grpc/src/main/proto/janusgraph/types/v1/schema_types.proto

```diff
@@ -84,6 +84,15 @@ message VertexLabel {
     repeated VertexProperty properties = 5;
 }
 
+message EdgeProperty {
+    // The Internal id.
+    google.protobuf.Int64Value id = 1;
```

Comment:
What's the difference with `int64` data type?

Reply:
If I'm correct, it allows the value to be nullable.

Reply:
There is a keyword `optional` which allows to not provide any value for a scalar type.
I.e.:
```
optional int64 id = 1;
```
I just don't know if there is any reason to use `google.protobuf.Int64Value` if we have native support for 64 bit values

Reply:
Before 3.15, Protobuf 3 hasn't support the keyword optional.

Reply:
I will do this refactoring in different PR.